Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Menu View class
+     *
+     * @module ez-usermenuview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_HIDDEN = 'is-user-profile-menu-hidden',
+        CLASS_USER_PROFILE = 'ez-user-profile',
+        SELECTOR_USER_PROFILE = '.' + CLASS_USER_PROFILE,
+        SELECTOR_USER_MENU_ITEM = '.ez-user-profile-menu-item',
```

Comment:
Nitpick: you could reuse previous variables here: `SELECTOR_USER_MENU_ITEM = SELECTOR_USER_PROFILE + '-item',`